src/app/showcase/showcase.component.html

```diff
@@ -1,2 +1,18 @@
-<a [routerLink]="['/', 'examples', exampleComponentName]" class="popout">Pop out »</a>
-<router-outlet></router-outlet>
\ No newline at end of file
+<div class="show-case">
+  <h1 class="show-case__header">{{exampleComponentName}}</h1>
+  <div class="showcase-cta-box" *ngIf="isCTABoxShown">
```

Comment:
"cta"?

Reply:
call to action forkortelse - naming is hard :)